### PR TITLE
ASoC: Intel: soc-acpi: mtl: add Dell SKU 0C64 and 0CC6

### PR DESCRIPTION
**Pull request by plbossart**
SKU 0C64 relies on rt713 (jack codec) on link0, rt1318 (single amplifier) on link2 and rt1713 (dmic) on link1.

SKU 0CC6 relies on rt713 (jack codec) on link0, rt1318 (two amplifiers) on link 1-2 and rt1713 (dmic) on link3.

### CODE REVIEW

**Comment by bardliao** [comment]
@plbossart For SKU0C64, 
```
        Device (SWD0)
        {
            Name (_ADR, 0x000331025D171301)  // _ADR: Address

        Device (SWD1)
        {
            Name (_ADR, 0x000031025D071301)  // _ADR: Address

        Device (SWD2)
        {
            Name (_ADR, 0x000130025D131801)  // _ADR: Address

```
So rt1713 (dmic) is on link 3, not link 1

**Comment by plbossart** [comment]
> So rt1713 (dmic) is on link 3, not link 1

And rt1318 on link1, not link2. I managed to have 2 devices wrong out of 3...
The ACPI folks put the devices in a different order, it's hard to parse all this :-)

Thanks for catching my mistake @bardliao, much appreciated.

**Comment by bardliao** [comment]
> > So rt1713 (dmic) is on link 3, not link 1
> 
> And rt1318 on link1, not link2. I managed to have 2 devices wrong out of 3... The ACPI folks put the devices in a different order, it's hard to parse all this :-)

Yeah, and the naming of `Device (SWD0)` is confusing. I usually think SDW0 means SDW link 0.

> 
> Thanks for catching my mistake @bardliao, much appreciated.

**Comment by plbossart** [comment]
@bardliao can you recheck the tables and approve if ok?

**Comment by bardliao** [comment]
> @bardliao can you recheck the tables and approve if ok?

@plbossart Have you pushed the change? I don't see any update.

**Comment by plbossart** [comment]
> @plbossart Have you pushed the change? I don't see any update.

my bad, forgot to do a git push. Updated now @bardliao

**Review by bardliao — APPROVED** [review]
